modules/sdk-coin-oas/src/oas.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @prettier
+ */
+import { BaseCoin, BitGoBase, common, MPCAlgorithm } from '@bitgo/sdk-core';
+import { BaseCoin as StaticsBaseCoin, coins } from '@bitgo/statics';
+import {
+  AbstractEthLikeNewCoins,
+  recoveryBlockchainExplorerQuery,
+  TransactionBuilder as EthLikeTransactionBuilder,
+} from '@bitgo/abstract-eth';
+import { TransactionBuilder } from './lib';
+
+export class Oas extends AbstractEthLikeNewCoins {
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Oas(bitgo, staticsCoin);
+  }
+
+  protected getTransactionBuilder(): EthLikeTransactionBuilder {
+    return new TransactionBuilder(coins.get(this.getBaseChain()));
+  }
+
+  /** @inheritDoc */
+  allowsAccountConsolidations(): boolean {
+    return true;
+  }
```

Comment:
This is not needed as it is having forwarders

Reply:
no need to add this method only as by default it is false in the BaseCoin, can remove in a follow up